### ISSUE TITLE
VALIDATION_ERROR - transactions[0].item_list.items[0].height

### ISSUE THREAD

**Issue by chkjamil** [paypal/PayPal-NET-SDK#96]
I am using following code for setting the item width, height and length

````csharp
new Item()
{
    name = "Igloo Scene - 1",
    description = "D1921",
    currency = "CAD",
    price = "10",
    quantity = "1",
    sku = "sku",
    width = new Measurement(){ unit = "meters", value = "4.7"},
    height = new Measurement(){ unit = "meters", value = "10"},
    length = new Measurement(){ unit = "meters", value = "7.5"}
}
````

however while creating a payment, it throw the below mentioned error for `width`, `height` and `length`

````
This field name is not defined for this resource type
````

The transaction is working fine after removing 

````csharp
width = new Measurement(){ unit = "meters", value = "4.7"},
height = new Measurement(){ unit = "meters", value = "10"},
length = new Measurement(){ unit = "meters", value = "7.5"}      
````

Kindly advice me how to send dimentions to paypal

Regards,

**Comment by jziaja** [comment]
Hi @chkjamil, great question!  I'm reaching out to the API team to find out why this is throwing a `VALIDATION_ERROR` and will circle back here when I have an answer.  Thanks!

**Comment by chkjamil** [comment]
Hi @jziaja , thank you for taking up this issue.

Looking forward.

**Comment by jziaja** [comment]
@chkjamil, it turns out these specific fields are meant to be internal-only for PayPal and should not have been included in the 1.4 release of this SDK.  We're working with the API team to identify any other properties/objects that were included in the 1.4 release, and I will mark these fields as obsolete in the upcoming 1.5 release.

For now, you can interpret these errors as the API not supporting these fields publicly.  

I should also mention that we are working on a newer process internally to avoid this miscommunication in future API & SDK releases.  Thanks again for bringing this to our attention. :+1:

**Comment by chkjamil** [comment]
Thanks @jziaja
